spec/features/staff/fund_managers_can_view_other_users_spec.rb

```diff
@@ -46,16 +50,20 @@
   end
 
   context "when the user is a delivery_partner" do
-    before { authenticate!(user: build_stubbed(:delivery_partner)) }
-
     scenario "cannot view the user" do
```

Comment:
Could we re-word this to better express what this test is doing? Something like 'does not see the mange users button on their organisation page'?